### ISSUE TITLE
[ZE] Colocar a dificuldade do mapa na votação.

### ISSUE THREAD

**Issue by gabrielacms** [quasemago/zrageservers#106]
Sugestão feita por: Matsu - https://steamcommunity.com/profiles/76561198130540435

Adicionar a dificuldade do mapa na frente do seu nome na votação. Para assim, tentar ajudar o pessoal a não se perder durante a votação. Essa dificuldade poderia ser representada pelas estrelinhas (que nem é no discord) ou escrito. 

-  Talvez se utilizasse as formas nominais de classificação ficaria mais fácil para eles entenderem (já que no servidor não terá legenda). Ex: Muito Fácil; Fácil; Médio; etc.

**Comment by lucaslinha** [comment]
Bom pela organização e informação.
Ruim pela contribuição de mapas overplayed, os que ja não tem motivação pra jogar outros mapas, não vão ter mesmo quando aparecer um medio ou dificil, a dificuldade vai assustar eles.

**Comment by hcjordao** [comment]
Concordo com o infused nesse ponto tbm. Mas podemos inserir no nominate para ver como ficaria

**Comment by alminha021** [comment]
Sou contra entrar na votação tambem, prefiro que tenham um incentivo a procurar um mapa e nomea-lo, antes e na votação os cara adoram reclamar de time, assim nao vejo muito mudança na realidade atual , vao votar em mapas manjados e faceis como prioridade.
quanto a colocar no nominate, ja ajuda bastante qm qr conhecer mapas , melhor que abrir ds pra ver a dificuldade do mapa  em #mapas-servidor , cara nomeado mapa que ele ache que o time possa ganhar, ele esta livre para nomear sabendo das limitações do time , podendo assim pedir para votarem no mapa que ele nomeou antes da votação.

**Comment by danielhidalgojunior** [comment]
> Concordo com o infused nesse ponto tbm. Mas podemos inserir no nominate para ver como ficaria

faço das suas palavras as minhas, meu rei

**Comment by lucaslinha** [comment]
Concordo com o alma, melhor por no nominate pro cara ter noção de cada mapa do que por na votaçao e acabar influenciando ela.

**Comment by danielhidalgojunior** [comment]
ze_frostdrake (Muito difícil)
ze_boatescape (Muito fácil)

o range poderia ser (muito fácil, fácil, mediano, difícil, muito difícil)

**Comment by alminha021** [comment]
Vai ser gerado outro problema de votação viciada , os cara nao vão jogar nunca mapa novo ou dificil.

Prefiro que mostre a dificuldade somente no nominate. 

Qualquer coisa que possa influenciar a votaçao pra mim eh errado, exceto em casos onde eh inserido algo para explicar algo que esta confuso.

Melhor identificar quem ta de usando nominate pra trolar e remover o nominate do cara temporariamente.
